### DISCUSSION TITLE
Is there a way to reference duckdb API documentation using  `intersphinx_mapping` ?

### DISCUSSION THREAD

**Discussion by jcfr** [duckdb/duckdb#12115] [Q&A] [answered]
This would allow to reference `duckdb` functions and objects from project documentation leveraging its API.

Attempting to use the following:

```python
intersphinx_mapping = {
    "duckdb": ("https://duckdb.org/docs/api/python/reference", None),
    "python": ("https://docs.python.org/3", None),
}
```

leads to the following:

```
WARNING: failed to reach any of the inventories with the following issues:
intersphinx inventory 'https://duckdb.org/docs/api/python/reference/objects.inv' not fetchable due to <class 'requests.exceptions.HTTPError'>: 404 Client Error: Not Found for url: https://duckdb.org/docs/api/python/reference/objects.inv
```

References:
* https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html

**Comment by kylebarron** [comment]
Agreed, this would be super nice. There are a lot of other projects building on top of duckdb, and being able to interlink to duckdb's Python API would be simpler for users.

**Comment by acirtep** [comment, marked as answer]
The objects.inv is available at: https://duckdb.org/docs/stable/clients/python/reference/objects.inv .

**Reply by krivard** [reply]
That URL appears to be 404ing now 😭

**Reply by kylebarron** [reply]
stable moved to current
